blosc/Cargo.toml

```diff
@@ -13,13 +13,9 @@ categories = ["api-bindings"]
 keywords = ["compression"]
 
 [dependencies]
-blosc-sys = "1.14.4"
+blosc-sys = { version = "1.14.5-pre", path = "../blosc-sys" }
```

Comment:
Yes, I should've done this all along.  I don't think that `version` is necessary when you're using `path`, though.  And it's one more thing to update when releasing a new version, so I'd rather leave it out.

Reply:
It is necessary to have `version` when publishing to `crates.io`. This is `semver`, and only needs to be updated on major releases